### DISCUSSION TITLE
What is the command to retrieve the text from the Labels -> Order number once user place order successfully ?

### DISCUSSION THREAD

**Discussion by shashijbs** [mobile-dev-inc/Maestro#1611] [General]
Hi Team,

How to retrieve text and save it in variable from Labels 
Ex : Once user placed successfully Cart and How to Retrieve Cart Order number and Save it in a variable ?

**Comment by shashijbs** [comment]
Could please help on this request - How to retrieve text from Label ? 

Ex : I want to retrieve text from label such as Order Number - Please help to share the command

**Comment by wenyuansong** [comment]
I had the same issue. 
Don't think Maestro supports anything dynamic in tests such as extract data from a page then input or assert on it and the JS it supports doesn't interact with DOM.  seems a major feature gap to make this tool useful.

**Comment by Fishbowler** [comment]
`copyTextFrom` seems like what folks are after here.

```
- copyTextFrom:
    text: 'Order: .*'
```

Of course, since it's a regular expression, you can be a bit more specific if you fancy, and get just the order number.

```
- copyTextFrom:
    text: 'AB\d{6}'
    label: 'Copy the order number (AB followed by 6 numbers)'
```
